lib/info.js

```diff
@@ -254,13 +254,9 @@ exports.getInfo = async (id, options) => {
   info.html5player = new URL(info.html5player, BASE_URL).toString();
 
   try {
-    if (options.playerClients.includes("WEB")) {
-      funcs.push(sig.decipherFormats(parseFormats(info.player_response), info.html5player, options));
-      funcs.push(...parseAdditionalManifests(info.player_response));
-    }
```

Comment:
Don't remove this. We can use `playerClients` option if you don't want to use this.